source/adminguide/networking/network_permissions.rst

```diff
@@ -0,0 +1,77 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information#
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+
+Guest Network Permissions
+-----------------------------
+
+From Apache CloudStack 4.17.0.0, guest networks can be shared to other
+accounts in the same domain by managing network permissions.
+
+The following networks can be shared:
```

Comment:
I think a new line at the end of this line would display the items below as a list, currently displayed in the same line:
<img width="735" alt="Screen Shot 2022-04-20 at 00 13 34" src="https://user-images.githubusercontent.com/5295080/164143229-ee617d24-0787-4997-8214-80779050072c.png">

Reply:
@nvazquez solved. thanks. now it looks like
![image](https://user-images.githubusercontent.com/57355700/164173812-8319b85f-6318-4b3a-812e-472d8b71add9.png)